examples/plugins/vite.config.integration.ts

```diff
@@ -1,21 +1,25 @@
 /// <reference types="vitest" />
-import { nxViteTsPaths } from '@nx/vite/plugins/nx-tsconfig-paths.plugin';
 import { defineConfig } from 'vite';
+import { tsconfigPathAliases } from '../../tools/vitest-tsconfig-path-aliases';
 
 export default defineConfig({
-  cacheDir: '../../node_modules/.vite/cli',
-  plugins: [nxViteTsPaths()],
+  cacheDir: '../../node_modules/.vite/examples-plugins',
   test: {
+    reporters: ['default'],
     globals: true,
     cache: {
       dir: '../../node_modules/.vitest',
     },
+    alias: tsconfigPathAliases(),
```

Comment:
the executor is missing this new property is the root cause?

Reply:
Previously, this was handled by `plugins: [nxViteTsPaths()]` (imported from `@nx/vite`). But after update it had no effect, only setting aliases works.